packages/strapi-plugin-i18n/tests/locales.test.e2e.js

```diff
@@ -306,11 +329,54 @@ describe('CRUD locales', () => {
     });
 
     test('Can delete a locale', async () => {
```

Comment:
You should split this into at least 2 tests: - simple locale delete & delete related entries